packages/cli/src/commands/build.js

```diff
@@ -93,24 +80,62 @@ export const handler = async ({
     )
   }
 
-  const tasks = new Listr(
-    side.map((sideName) => {
-      const { cwd, cmd } = execCommandsForSides[sideName]
-      return {
-        title: `Building "${sideName}"${(stats && ' for stats') || ''}...`,
+  const listrTasks = side.map((sideName) => {
+    const { cwd, cmd } = execCommandsForSides[sideName]
+    return {
+      title: `Building "${sideName}"${(stats && ' for stats') || ''}...`,
+      task: () => {
+        return execa(cmd, undefined, {
+          stdio: verbose ? 'inherit' : 'pipe',
+          shell: true,
+          cwd,
+        })
+      },
+    }
+  })
+
+  // Additional tasks, apart from build
+  if (side.includes('api')) {
+    try {
+      await generatePrismaClient({
+        verbose,
+        force: true,
+        schema: getPaths().api.dbSchema,
+      })
+    } catch (e) {
+      console.log(c.error(e.message))
+      process.exit(1)
+    }
+  }
+
+  if (side.includes('web')) {
+    if (getConfig().web.experimentalPrerender) {
+      listrTasks.push({
+        title: 'Prerendering your RW app...',
         task: () => {
-          return execa(cmd, undefined, {
+          return execa('yarn rw prerender', undefined, {
             stdio: verbose ? 'inherit' : 'pipe',
             shell: true,
-            cwd,
+            cwd: getPaths().base,
           })
         },
-      }
-    }),
-    {
-      renderer: verbose && VerboseRenderer,
+      })
     }
-  )
+    listrTasks.unshift({
```

Comment:
This code might be easier to read if you place it above the pre-render block and just use push.

Reply:
Updating this now, but we not using push because clean needs to run before the web builds!